cmds/chmod/chmod_test.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+)
+
+var (
+	testPath = "."
+	// if true removeAll the testPath on the end
+	remove = true
+)
+
+const (
+	maxSizeFile = 4096
+	rangeRand   = 100
+	maxDirDepth = 5
+	maxFiles    = 5
+)
+
+// randomFile create a random file with random content
+func randomFile(fpath, filename string) (*os.File, error) {
+	f, err := ioutil.TempFile(fpath, filename)
+	if err != nil {
+		return nil, err
+	}
+	// generate random content for files
```

Comment:
I don't think you need to add content. It will add system calls we don't want, take time, and this is chmod after all.